docs/Lama2/docs/reference/preprocess.md

```diff
@@ -67,7 +67,7 @@ func GetLamaFileAsString(path string) string
 func LamaFile(inputFile string) (string, string)
 ```
 
-LamaFile takes in a path to an API file. It moves into the API file directory, reads the API contents, loads the \`l2.env\` file if available, and finally substitutes environment vars in the API contents Once done, it reverts back to the original directory, and returns the processed l2 file.
+LamaFile takes in a path to an API file. It moves into the API file directory, reads the API contents, loads the \`l2config.env\` global variables file if available in any of parent directory and \`l2.env\` file if available in present directory , and finally substitutes environment vars in the API contents Once done, it reverts back to the original directory, and returns the processed l2 file.
```

Comment:
Comment can make the "overwriting" part more obvious. Sentence is too long now, maybe simplify a bit.